src/gui/optionsdialog.ui

```diff
@@ -839,6 +839,56 @@
                  </property>
                 </widget>
                </item>
+               <item>
+                <layout class="QHBoxLayout" name="stopConditionLayout">
+                 <item>
+                  <widget class="QLabel" name="stopConditionLabel">
+                   <property name="text">
+                    <string>Torrent stop condition:</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QComboBox" name="stopConditionComboBox">
+                   <property name="currentIndex">
+                    <number>0</number>
+                   </property>
+                   <property name="toolTip">
+                    <string>Metadata received: Torrent will stop after metadata is received. Torrents that have metadata initially aren't affected.
```

Comment:
I would still add the `None` case for completeness:
`The torrent will not stop and continue to download as usual`

Reply:
`None` isn't a "stop condition". It just means that no "stop condition" is set.

Reply:
>It just means that no "stop condition" is set.

Right, and you should explain it in tooltip when you are using one.